my-app/src/SortResults.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Child Class 2 - renders dropdown for selecting sort category  
+ * Stateless Component Class (could be turned into a function component )
+ */
+import React from 'react';
+
+export const SortResults = (props) => {
+
+    console.log("2. I let you decide how to sort results");
+    let searchStatus = props.searchStatus; 
+
+    if (searchStatus) {  
+
+      return (
+        <div>
+          <br/>
+          <label htmlFor="sort"> Sort by: </label>
+          <select id="sort" name="sort" onChange={props.updateSortCriteria}>
+            <option value="Title">Title A-Z</option>
+            <option value="Year">Year (Oldest - Newest)</option>
+            <option value="imdbID">imdbID no.</option>
+          </select>
+        </div>
+      )
+  } else {
+    return null;
```

Comment:
Is it ok to return null in React, or does the `render()` method need to have some element returned?

Reply:
Yes we can return an empty value from a React render method. According to the docs: false , null , undefined , and true are valid children. They simply don't render. 

This is useful for conditional rendering, especially now we have split out conditions into seperate components. Alternatively we can return <></>, but returning null is a good practice, when returning <></> react will need to do extra work.